blocks_vertical/data.js

```diff
@@ -160,7 +160,8 @@ Blockly.Blocks['data_listcontents'] = {
         {
           "type": "field_variable_getter",
           "text": "",
-          "name": "LIST"
+          "name": "VARIABLE",
```

Comment:
These are all list blocks.  Why are you renaming their fields to VARIABLE instead of LIST?